crates/runtime/src/dataconnector.rs

```diff
@@ -375,11 +385,12 @@ pub trait ListingTableConnector: DataConnector {
                 dataconnector: "ListingTableConnector",
             })?;
 
-        let table = ObjectStoreMetadataTable::try_new(object_store, Some(prefix_str), Some(filename_regex))
-            .context(InternalSnafu {
-                dataconnector: "ListingTableConnector",
-                code: "LTC-MP-OSMT".to_string(), // ListingTableConnector-MetadataProvider-ObjectStoreMetadataTableTryNew
-            })?;
+        let table =
+            ObjectStoreMetadataTable::try_new(object_store, Some(prefix_str), Some(filename_regex))
+                .context(InternalSnafu {
+                    dataconnector: "ListingTableConnector",
```

Comment:
```suggestion
                    dataconnector: format!("{self}"),
```

Reply:
Thank you. Before I had `where Self: Display` i couldn't use `format!("{self}")`